tests/sites_tests/tests.py

```diff
@@ -93,6 +93,16 @@ def test_get_current_site_no_site_id(self):
         site = get_current_site(request)
         self.assertEqual(site.name, "example.com")
 
+    @override_settings(SITE_ID='', ALLOWED_HOSTS=['example.com'])
+    def test_get_current_site_no_site_id_host_with_trailing_dot(self):
+        request = HttpRequest()
+        request.META = {
+            "SERVER_NAME": "example.com.",
```

Comment:
Use single quotes please. (We use single quotes if the string itself doesn't contain them, existing code isn't consistent in this convention.)